src/front/spv.rs

```diff
@@ -330,33 +365,46 @@ struct LookupSampledImage {
     image: Handle<crate::Expression>,
     sampler: Handle<crate::Expression>,
 }
-
 struct DeferredFunctionCall {
     source_handle: Handle<crate::Function>,
     expr_handle: Handle<crate::Expression>,
     dst_id: spirv::Word,
 }
-
-enum Terminator {
+pub enum Merge {
+    Selection {
+        merge_block_id: spirv::Word,
```

Comment:
how do you feel about doing something like this:
```rust
type BlockId = std::num::NonZeroU32;
```
If this is used in all the places where you have `Option<spirv::Word>` (and the Word is a block ID), we'd save a bunch of bytes, and make the data cache utilization better

Reply:
I love this.